tests/modules/token_embedders/pretrained_transformer_embedder_test.py

```diff
@@ -330,3 +330,22 @@ def test_embeddings_resize(self):
             enhanced_token_embedder.transformer_model.embeddings.word_embeddings.num_embeddings
             == 28997
         )
+
+    def test_eval_mode(self):
+        token_embedder = PretrainedTransformerEmbedder("bert-base-uncased", train_parameters=False)
```

Comment:
You could use the "epwalsh/bert-xsmall-dummy" model instead of "bert-base-uncased" to make this test more lightweight.

Reply:
```suggestion
        token_embedder = PretrainedTransformerEmbedder("epwalsh/bert-xsmall-dummy", train_parameters=False)
```